page.go

```diff
@@ -40,32 +40,36 @@ type page struct {
 	// pageStatus is set to '4' if the transaction has been committed and
 	// applied, meaning that the transaction can be ignored upon load, and the
 	// associated pages can be reclaimed.
-	pageStatus uint64 // Gets marshalled to disk.
+	pageStatus uint64 // marshalled to disk.
 
 	// nextPage points to the logical next page in the logFile for this
 	// transaction. The page may not appear in the file in-order. This value
 	// is set to math.MaxUint64 if there is no next page (indicating this
 	// page is the last). The first page may be the only page, in which
 	// case it is also the last page.
-	nextPage *page // Gets marshalled as nextPage.offset.
+	nextPage *page // marshalled as nextPage.offset.
 
 	// transactionNumber is only saved for the last page, which can be
 	// determined by nextPage being nil, or when marshalled to disk if
 	// it is marshalled to math.MaxUint64
-	transactionNumber uint64 // Gets marshalled to disk.
+	transactionNumber uint64 // marshalled to disk.
 
 	// payload contains the marshalled update, which may be spread over multiple
 	// pages if it is large. If spread over multiple pages, the full payload
 	// can be assembled by appending the separate payloads together. To keep the
 	// full size of the page at under pageSize bytes, the payload should not
 	// be greater than (pageSize - 64 bytes).
-	payload []byte // Gets marshalled to disk.
+	payload []byte // marshalled to disk.
```

Comment:
If we are being super pedantic these should either be `Marshalled to disk.` or `marshalled to disk` with preference for the former

Reply:
(non-blocking issue)